### ISSUE TITLE
Spotlightlight search results icon

### ISSUE THREAD

**Issue by AshUK** [jaretburkett/electron-icon-maker#8]
![image](https://user-images.githubusercontent.com/1187198/39826591-231082fa-53ad-11e8-8068-fa9a8dcff40f.png)

The icon does not display correctly in spotlight results

**Comment by Kilian** [comment]
This is due to electron-icon-maker not adding the correct sizes into the icns file. It should contain only 16x16, 32x32, 128x128, 256x256, 512x512 and 1024x1024 pixels

You could use https://iconverticons.com/ in the mean time, or `icnsutils`

**Comment by kcak11** [comment]
will this issue be fixed sooner or later ? I recently used the electron-icon-maker and noticed the same issue as described above. It would be good if the author takes sometime to fix this.

Thanks in advance.

Related issue: https://github.com/electron-userland/electron-packager/issues/929

**Comment by lapagept** [comment]
Too bad! I have the same problem. I liked electron-icon-maker...

**Comment by crossle** [comment]
same problem

**Comment by voidgraphics** [comment]
I tried to change this package's source to only include the sizes mentioned by @Kilian but it still came out corrupted. Not sure why.

For anyone struggling with this, here's how I managed to generate my icons properly. I used the `iconutil` tool which should be preinstalled on your mac.

1. Make a directory with the `.iconset` extension and put your pngs in it. They should be named like so: `icon_16x16.png`
2. run `iconutil -c icns yourdirectory.iconset`
3. It should create `yourdirectory.icns`

If you're not on mac, I think I saw iconutil on NPM. Not sure if it's the same tool exactly but worth a try.
